theano/gof/cmodule.py

```diff
@@ -1584,7 +1584,12 @@ def std_include_dirs():
     python_inc_dirs = ([py_inc] if py_inc == py_plat_spec_inc
                        else [py_inc, py_plat_spec_inc])
     gof_inc_dir = os.path.abspath(os.path.dirname(__file__))
-    return numpy_inc_dirs + python_inc_dirs + [gof_inc_dir]
+    other_dirs = []
+    if sys.platform == 'win32':
+        alt_inc_dir = os.path.abspath(os.path.normpath(sys.exec_prefix + '\Library\include'))
+        if os.path.exists(alt_inc_dir) and os.path.isdir(alt_inc_dir):
+            other_dirs.append(alt_inc_dir)
+    return numpy_inc_dirs + python_inc_dirs + [gof_inc_dir] + other_dirs
```

Comment:
I would rather place this hack in GpuArrayType than here.  That way we won't invoke it for things that don't require the new backend.